ads/rubicon.md

```diff
@@ -100,21 +71,6 @@ By default the ad size is based on the `width` and `height` attributes of the `a
 - `data-zone`
```

Comment:
should we include `type` in this list of attributes?  I see it in the `ads-legacy.amp.html` and `ads.amp.html` examples, as well as `validator/testdata/feature_tests/ads.[html|out]`

Reply:
See answer for other question.